FirebaseRemoteConfig/Sources/RCNConfigRealtime.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef RCNConfigRealtime_h
+#define RCNConfigRealtime_h
+
+#import <Foundation/Foundation.h>
+#import "FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h"
+
+@interface RCNConfigRealtime : NSObject <NSURLSessionDataDelegate>
+
+@property(strong, atomic, nonnull) NSMutableDictionary<NSString *, id> *listeners;
+@property(strong, atomic, nonnull) dispatch_queue_t lockQueue;
```

Comment:
Rename the queue to something meaningful like listenersSerialQueue

Reply:
Also all the private properties should probably move to RCNConfigRealtime.m file instead of RCNConfigRealtime.h.

Like this way: https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseMessaging/Sources/FIRMessaging.m#L84

Reply:
Thanks for the review! Updated.